### PR TITLE
Feature/add seeds 007 through 012

### PR DESCRIPTION
**Pull request by DomenicScarcella**
Description
What work was done?
• Created seeds files 007 through 012 for seeds folder
Why was this work done?
• Getting seeds files to match the new migration files
Type of change
• New feature (non-breaking change which adds functionality)
Change Status
• Completed, ready to review and merge
Checklist
• My code follows the style guidelines of this project
• I have performed a self-review of my own code
• My changes generate no new warnings
• There are no merge conflicts

### CODE REVIEW

File: data/seeds/007-service_type_providers.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+const service_type_providers = [
+  {
+    service_type_id: 1,
+    provider_id: 'd49e0c39-010f-481c-862a-df8c193ed7ac',
```
Comment (jaypdl) [inline data/seeds/007-service_type_providers.js:4]: Due to new revelations relating to Okta, for provider_id can you switch to one these:

00unr48onuAmU9sxK5d6
00unr8nm2sJkxkcrH5d6
00unr5lzndqVF13jP5d6
00uk9lxaulDYOiB4H5d6
00unr3s3m2zHx70ck5d6

Comment (jaypdl) [reply data/seeds/007-service_type_providers.js:4]: instead of the uuids you used. This change is only for places where provider_id is used

**Review by jaypdl — CHANGES_REQUESTED** [review]
Looks good! Could you just change the provider_id as mentioned in the comment?

File: data/seeds/007-service_type_providers.js (hunk 1 above)
Comment (DomenicScarcella) [reply data/seeds/007-service_type_providers.js]: Edits made and file pushed.

File: data/seeds/009-households.js [hunk 2]
```diff
@@ -0,0 +1,152 @@
+const faker = require('faker');
+
+const getRand = (max) => {
```
Comment (jaypdl) [inline data/seeds/009-households.js:3]: Good job with the randomizer!

File: data/seeds/009-households.js [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+  return Math.floor(Math.random() * max) + 1;
+};
+
+const households = [
+  {
+    household_id: '1b9efd46-0a29-4ed0-9646-4eb01fcb1a78',
+    household_name: faker.fake(`{{name.lastName}}`),
```
Comment (jaypdl) [inline data/seeds/009-households.js:10]: I like how you used a fake data service!

File: data/seeds/007-service_type_providers.js [hunk 4]
```diff
@@ -0,0 +1,42 @@
+const service_type_providers = [
+  {
+    service_type_id: 1,
+    provider_id: '00unr48onuAmU9sxK5d6',
```
Comment (jaypdl) [inline data/seeds/007-service_type_providers.js:4]: Thanks for making the provider_id changes I requested!

**Review by jaypdl — APPROVED** [review]
Looks good! Thanks for making the changes! Approved!

**Review by MatthewEllisTx — APPROVED** [review]

**Pull request merged**